pwndbg/commands/heap.py

```diff
@@ -611,13 +617,20 @@ def find_fake_fast(addr, size=None):
     default=False,
     help="Attempt to keep printing beyond the top chunk.",
 )
+parser.add_argument(
+    "--all",
+    "-a",
+    action="store_true",
+    default=False,
+    help="Display all the chunk contents (Ignore the `max-visualize-chunk-size` configuration).",
+)
 
 
 @pwndbg.commands.ArgparsedCommand(parser)
 @pwndbg.commands.OnlyWhenRunning
 @pwndbg.commands.OnlyWithResolvedHeapSyms
 @pwndbg.commands.OnlyWhenHeapIsInitialized
-def vis_heap_chunks(addr=None, count=None, naive=None):
+def vis_heap_chunks(addr=None, count=None, naive=None, all=None):
```

Comment:
`all` is a Python built-in, the linter will reject this in the near future. Can you rename to something else? If `all` is the best name, I'm fine with `all_`.

Reply:
changed to `display_all`